nbbuild/cluster.properties

```diff
@@ -33,7 +33,8 @@ clusters.config.release.list=\
 
 # full config must contain all clusters
 clusters.config.full.list=\
-        ${clusters.config.release.list}
+        ${clusters.config.release.list},\
+        nb.cluster.rust
```

Comment:
just as note: I hope that this works. This would put rust behind the ergonomics cluster. If this causes problems we have to swap the two, or copy the list from the `release.list`, but add rust just before ergonomics.

Reply:
Very true, let's place `nb.cluster.rust` before `${clusters.config.release.list}` in a new commit.